File: rest/notification/list-get-example-3/list-get-example-3.6.x.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import Client
from datetime import date

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token = "your_auth_token"
client = Client(account_sid, auth_token)

# A list of notification objects with the properties described above
notifications = client.notifications \
                      .list(message_date_after=date(2009, 7, 6), log="1")

for notification in notifications:
    print(notification.request_url)

File: rest/usage-records/list-get-example-3/list-get-example-3.6.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account.Usage;

class Example
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Auth Token at twilio.com/console
        // To set up environmental variables, see http://twil.io/secure
        const string accountSid = Environment.GetEnvironmentVariable("TWILIO_ACCOUNT_SID");
        const string authToken = Environment.GetEnvironmentVariable("TWILIO_AUTH_TOKEN");
        TwilioClient.Init(accountSid, authToken);

        var records = RecordResource.Read(
            category: RecordResource.CategoryEnum.CallsInbound,
            startDate: new DateTime(2012, 09, 01),
            endDate: new DateTime(2012, 09, 30));

        foreach (var record in records)
        {
            Console.WriteLine(record.Price);
        }
    }
}
